webpack/js/main.js

```diff
@@ -1,74 +1,112 @@
 $(document).ready(function () {
-    $(window).scroll(function () {
-        if ($(this).scrollTop() > 500) {
-            $('#back-to-top').fadeIn();
-        } else {
-            $('#back-to-top').fadeOut();
-        }
-    });
+  $(window).scroll(function () {
+    if ($(this).scrollTop() > 500) {
+      $('#back-to-top').fadeIn();
+    } else {
+      $('#back-to-top').fadeOut();
+    }
+  });
 
-    patchAssetIntoDom('/assets/logos/cc/logomark.svg');
-    patchAssetIntoDom('/assets/logos/products/open_source.svg');
+  patchAssetIntoDom('/assets/logos/cc/logomark.svg');
+  patchAssetIntoDom('/assets/logos/products/open_source.svg');
 
-    // Check for click events on the navbar burger icon
-    $(".navbar-burger").click(function() {
-        // Toggle the "is-active" class on both the "navbar-burger" and the "navbar-menu"
-        $(".navbar-burger").toggleClass("is-active");
-        $(".navbar-menu").toggleClass("is-active");
-    });
+  // Check for click events on the navbar burger icon
+  $(".navbar-burger").click(function () {
+    // Toggle the "is-active" class on both the "navbar-burger" and the "navbar-menu"
+    $(".navbar-burger").toggleClass("is-active");
+    $(".navbar-menu").toggleClass("is-active");
+  });
 
-    $(".navbar-item.has-dropdown").click(function() {
-        if ($(".navbar-burger").is(':visible')) {
-            $(this).toggleClass("is-active");
-        }
-    });
-    $(".navbar-item > .navbar-link").click(function(e) {
-        if ($(".navbar-burger").is(':visible')) {
-            e.preventDefault();
-        }
-    });
-    $(window).resize(function() {
-        if (!$(".navbar-burger").is(':visible') && $(".navbar-item.has-dropdown.is-active").length) {
-            $(".navbar-item.has-dropdown.is-active").removeClass('is-active');
-        }
+  $(".navbar-item.has-dropdown").click(function () {
+    if ($(".navbar-burger").is(':visible')) {
+      $(this).toggleClass("is-active");
+    }
+  });
+  $(".navbar-item > .navbar-link").click(function (e) {
+    if ($(".navbar-burger").is(':visible')) {
+      e.preventDefault();
+    }
+  });
+  $(window).resize(function () {
+    if (!$(".navbar-burger").is(':visible') && $(".navbar-item.has-dropdown.is-active").length) {
+      $(".navbar-item.has-dropdown.is-active").removeClass('is-active');
+    }
+  });
+
+  $(window).scroll(function () {
```

Comment:
This could be very bad for performance. Maybe set [listener to `passive`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Parameters), or even better use [the Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API)?

Reply:
Cool.